docs/inbound-agent.md

```diff
@@ -78,7 +78,7 @@ This mechanism requires a download of the `agent.jar`, as described for "Downloa
 
 Once all the prerequisite files and data have been obtained, the agent can be launched with a command like this
 ```
-java -cp agent.jar hudson.remoting.jnlp.Main \
+java -jar agent.jar \
   -workDir <work directory> \
   -direct <HOST:PORT> \
   -protocols JNLP4-connect \
```

Comment:
I think https://github.com/jenkinsci/remoting/blob/2232d9d9beb25ac293aa6bb2eb67ab097c87d173/docs/inbound-agent.md#L86 should now read

```
-secret <secretString> -name <agentName>
```

or similar to avoid using the deprecated positional arguments?

Reply:
Fixed in commit 7875c0d0